dist/kafka-sampler/config.yaml

```diff
@@ -13,16 +13,23 @@ kafka:
   # Convert field names to all lowercase and divide nested structs with `_`
   # sarama:  (...)
 
-  # If unset, it will create a `Sampler` per each topic found in the Kafka cluster,
-  # supports regex RE2 syntax as described at https://github.com/google/re2/wiki/Syntax, except for `\C`.
-  # It always ignores internal topics like: `__consumer_offsets` and `__transaction_state`.
-  # topicfilter: 
+  # Topics contains all the configuration about the topics that have to be read from
+  # topics:
+  #   # Maximum number of topics to read from (safeguard). Topics after the limit is reached
```

Comment:
I would say it a bit more plainly:
`Safeguard to avoid consuming an unexpectedly large number of topics. The first `max` number of topics that match the filter rules (in no particular order) will be monitored, the rest will be ignored.`